lighthouse-cli/manual-chrome-launcher.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+'use strict'
+
+require('./compiled-check.js')('chrome-launcher.js');
+
+const args = process.argv.slice(2);
+
+let chromeArgs;
+let startingUrl;
+
+if (args.length) {
+  // If the first argument starts with -- assume all are chrome args.
+  if (args[0].startsWith('--')) {
```

Comment:
This assumption is a little restrictive IMO.. Launching chrome as `chrome --flags http://url` is the most typical way, so I think we should preserve that.

Let's send everything that `startsWith('-')` as `additionalFlags` and send any remaining args as the startingURL string. 
(Technically multiple URLs will open multiple tabs and that should work already via chromeLauncher)